src/awkward/nplike.py

```diff
@@ -422,6 +420,60 @@ def __call__(self, *args):
         )
 
 
+class CupyKernel(NumpyKernel):
+    def max_length(self, args):
+        cupy = ak._v2._connect.cuda.import_cupy("Awkward Arrays with CUDA")
+        max_length = numpy.iinfo(numpy.int64).min
```

Comment:
This is why you want that to be a Python constant. It's showing up everywhere.

Reply:
This is for a different use-case.

Reply:
Aha! You're right, it is. (That's more reason for wanting the `NO_ERROR` to look different.)